src/core/document.js

```diff
@@ -299,6 +299,8 @@ var Page = (function PageClosure() {
           resources: this.resources,
           normalizeWhitespace,
           combineTextItems,
+          sink,
+          seenStyles: Object.create(null),
```

Comment:
Yes, sure. We can do this way also. I think i get confused by the https://github.com/mozilla/pdf.js/pull/8488#issuecomment-306547988